crates/forge/tests/data/collection_without_lib/Scarb.toml

```diff
@@ -1,9 +1,12 @@
 [package]
 name = "collection_without_lib"
 version = "0.1.0"
-edition = "2023_10"
+edition = "2024_07"
 
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 
 [dependencies]
 # foo = { path = "vendor/foo" }
+
+[dev-dependencies]
+snforge_std = { path = "../../../../../snforge_std" }
```

Comment:
Is this needed (same for other additions of snforge_std)? It seems it's not used.